src/Http/routes.php

```diff
@@ -27,6 +29,12 @@
 Route::post('/api/pages/{id}', 'PagesController@store')->name('pages.store');
 Route::delete('/api/pages/{id}', 'PagesController@delete')->name('pages.delete');
 
+// Categories...
+Route::get('/api/categories', 'CategoriesController@index')->name('categories.index');
+Route::get('/api/categories/{id?}', 'CategoriesController@show')->name('categories.show');
```

Comment:
If id isn't provided on this route it will match the index route from the row above. That is why I think it shouldn't be optional.